packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -44,38 +32,27 @@ The `@customElement` decorator defines a custom element, equivalent to calling:
 customElements.define('my-element', MyElement);
 ```
 
-The `@property` decorator declares a reactive property.
+The `@property` decorator declares a [reactive property](/docs/v3/components/properties/).
 
-See [Reactive properties](/docs/v3/components/properties/) for more information about configuring properties.
+Other decorators provided by the Lit package can be found in the list of [built-in decorators](#built-in-decorators).
 
-## Built-in decorators
+## Background
```

Comment:
I wonder if this is important enough to come before the using sections? Maybe it could go down but have a more descriptive title? Like "Decorators versions and standards"?